code/game/machinery/cryopod.dm

```diff
@@ -192,12 +192,13 @@
 		/obj/item/device/paicard,
 		/obj/item/weapon/gun,
 		/obj/item/weapon/pinpointer,
-		/obj/item/clothing/suit,
```

Comment:
RD's armor, NASA suits and all the unique space suits which you can't make or order from cargo are out with this.